index.html

```diff
@@ -0,0 +1,772 @@
+            <span class="original_price">$2000</span>
+            <span class="new_price">$29</span>
+        </div>
+    </div>
+    <div class="section5_items">
+        <img src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdnp3.stackassets.com/b34f2af9e9ec3122eb0c0b252532b61f8a08c0ea/store/243275a3bf8c7d20203d2e6e3b9f3664926f9f8913e755d1ae8811aecd9b/sale_25748_primary_image.jpg"
+            alt="">
+        <div>
+            <h4><a href="#"> The Complete 2021 Microsoft Azure Certification Prep Bundle</a></h4>
+            <span class="original_price">$1839</span>
+            <span class="new_price">$39</span>
+        </div>
+    </div>
+    <div class="section5_items">
+        <img src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdnp3.stackassets.com/549384f77e88448bc914df09683a756f0c5e4c57/store/70bd980cd752af445b083eb46b1db2fb3fcee3c55eddbade2cae4193f5c5/sale_162254_primary_image.jpg"
+            alt="logicpro">
+        <div>
+            <h4><a href="#"> IPVanish VPN: 1-Yr Subscription</a></h4>
+            <span class="original_price">$143 </span>
+            <span class="new_price">$29</span>
+        </div>
+    </div>
+    <div class="section5_items">
+        <img src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdnp3.stackassets.com/131a74f7d56643b61ee30fb3e8da705c53031bed/store/b32bf58c9865347773f96c04f56c69f07227e61baeb48f8ffffec4861e5e/sale_18296_primary_image.jpg"
+            alt="logicpro">
+        <div>
+            <h4><a href="#"> The Ultimate Logic Pro X Music Production Bundle</a></h4>
+            <span class="original_price">$690 </span>
+            <span class="new_price">$29</span>
+        </div>
+    </div>
+    <div class="section5_items">
+        <img src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdnp2.stackassets.com/dd2a99b3f7d21cd700c3d210798ff74322c011d5/store/f4313846701bed0b9cbd6b0a983722fdce21faef621975d07c3a90a79e55/sale_148067_primary_image.jpg"
+            alt="logicpro">
+        <div>
+            <h4><a href="#"> The 2021 Complete Microsoft Azure Certification Prep Bundle</a></h4>
+            <span class="original_price">$1194 </span>
+            <span class="new_price">$34.99</span>
+        </div>
+    </div>
+    <div class="section5_items">
+        <img src="https://wr.perma-archives.org/public/m5zv-q2d6/20210126152647mp_/https://cdnp1.stackassets.com/78d21e7906585957a1c4878636e0265d8bc3dffe/store/b39642357ca106ff7341d682ee27edcec23cbb3b7c2e9ddc690bb89533f1/sale_162717_primary_image.jpg"
+            alt="logicpro">
+        <div>
+            <h4><a href="#"> Familio Private Backup Platform: Lifetime Subscription</a></h4>
+            <span class="original_price">$300 </span>
+            <span class="new_price">$39.99</span>
+        </div>
+    </div>
+</section> -->
+
+  <footer class="tnw-foot">
+    <div class="foot-icn">
```

Comment:
All social media icons should be enclosed in a `nav` tag. Each icon should be enclosed in an `a` tag.